packages/compartment-mapper/NEWS.md

```diff
@@ -4,6 +4,10 @@ User-visible changes to `@endo/compartment-mapper`:
 
 - Omits unused module descriptors from `compartment-map.json` in archived
   applications, potentially reducing file sizes.
+- Fixes an issue where errors thrown from exit module hooks (`importHook`) would
+  be thrown at parse-time instead of at runtime. Such errors will now be thrown
+  at runtime, as expected. To those who expected the previous behavior: if you
+  exist, please exercise caution when upgrading.
```

Comment:
Should be qualified that the behavior only applies to parsers that have heuristic import analysis. The behavior for ESM should be to throw an error while loading if a static import dependency is unmet.

Reply:
OK, I noted this